omnisharp.json

```diff
@@ -0,0 +1,7 @@
+{
+  "FormattingOptions": {
+    "UseTabs": true,
+    "SpaceAfterCast": true,
+    "IndentSwitchCaseSection": false
+  }
+}
```

Comment:
Consider moving formatting settings to `.editorconfig` and opting OmniSharp into using that: https://www.strathweb.com/2019/07/editorconfig-support-in-omnisharp-and-c-extension-vs-code/

Reply:
Oh great, I didn't know they finally shipped that. It's surprising it's not on by default.

Reply:
I, too, was surprised.